src/electron/README.md

```diff
@@ -30,6 +29,26 @@ npm run action electron/start [windows|linux]
 
 ### Windows
 
+To build for Windows on a macOS, you need to install minGW v11.0.1.
+
+With Homebrew (how to ensure consistent version?):
```

Comment:
Will MacPorts allow you downloading an old version? Actually homebrew is not the official release channel, MacPorts is: https://www.mingw-w64.org/downloads/

Reply:
Yes, updated.